functions/views/prescription-template.ejs

```diff
@@ -59,14 +59,16 @@
                 <% if (details) { %>
                     <div class="info"> 
 			         <p><span><%= details.doctor.name %></span></p>
-			         <p><%= details.doctor.designation %></p>
+                     <p><%= details.doctor.designation %>, <%= details.doctor.specialization %></p> 
```

Comment:
Please don't add specialization for now.
Designation is fine.

Reply:
Done.

Reply:
Nvm, this is fine :)